jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/resources/behavior/EndToEndBehavior.java

```diff
@@ -30,20 +30,20 @@
 
 public class EndToEndBehavior {
 
+  private static final Configuration cfg = new Configuration(
```

Comment:
nitpick: This doesn't really need to be a static field because nothing is writing to it